types/react-intl-redux/index.d.ts

```diff
@@ -20,4 +20,4 @@ interface IntlAction extends Action {
 export function intlReducer(state: IntlState, action: IntlAction): IntlState
 export function updateIntl (opts: IntlState): IntlAction
 export class IntlProvider extends ReactIntlProvider {}
-export class Provider extends ReduxProvider {}
+export class Provider<A extends IntlAction> extends ReduxProvider<A> {}
```

Comment:
It might be necessary to add a type parameter default here.

Reply:
This is supposed to be inferred too, from the `store` prop.